### ISSUE TITLE
Question about configuration 

### ISSUE THREAD

**Issue by PhNeu23** [zihuazheng/DIP#10]
First of all, thank you very much for your great work. 
I would like to reproduce the values for Sintel (Train) and Kitti (Train) from your paper (Table 4). Could you tell me what configuration you used (number of iters, warm/cold start etc.) to get these values ?

**Comment by zihuazheng** [comment]
> First of all, thank you very much for your great work. I would like to reproduce the values for Sintel (Train) and Kitti (Train) from your paper (Table 4). Could you tell me what configuration you used (number of iters, warm/cold start etc.) to get these values ?
Thank you for your interest in our work.
In Table 4, we consistently used 12 iterations throughout the experiments. Specifically, RAFT's warm-up strategy was employed during the evaluation on the Sintel dataset, while random initialization was performed at the start of each evaluation on the Kitti dataset

**Comment by zihuazheng** [comment]
> First of all, thank you very much for your great work. I would like to reproduce the values for Sintel (Train) and Kitti (Train) from your paper (Table 4). Could you tell me what configuration you used (number of iters, warm/cold start etc.) to get these values ?

Thank you for your interest in our work.
In Table 4, we consistently used 12 iterations throughout the experiments. Specifically, RAFT's warm-up strategy was employed during the evaluation on the Sintel dataset, while random initialization was performed at the start of each evaluation on the Kitti dataset

**Comment by zihuazheng** [comment]
> First of all, thank you very much for your great work. I would like to reproduce the values for Sintel (Train) and Kitti (Train) from your paper (Table 4). Could you tell me what configuration you used (number of iters, warm/cold start etc.) to get these values ?

In Table 4, as outlined in the paper, we consistently employ random initialization during training. For fine-tuning on the Sintel dataset, we combine data from Sintel and FlyingThings3D and train with a size of 768 × 384 for 100k iterations. Additionally, we conduct fine-tuning using a combination of data from FlyingThings, Sintel, KITTI-15, and HD1K for 100k iterations with a training size of 832 × 320. The number of GRU iterations during training is 6.

**Comment by PhNeu23** [comment]
Thank you for the response to my question. This will help me to reproduce the results.
